src/algorithms/sfx/tctototal.cpp

```diff
@@ -28,8 +28,8 @@ const char* TCToTotal::name = "TCToTotal";
 const char* TCToTotal::category = "Envelope/SFX";
 const char* TCToTotal::description = DOC("This algorithm calculates the ratio of the temporal centroid to the total length of a signal envelope. This ratio shows how the sound is 'balanced'. Its value is close to 0 if most of the energy lies at the beginning of the sound (e.g. decrescendo or impulsive sounds), close to 0.5 if the sound is symetric (e.g. 'delta unvarying' sounds), and close to 1 if most of the energy lies at the end of the sound (e.g. crescendo sounds).\n"
 "\n"
-"Please note that the TCToTotal ratio is not defined for a zero signal (a signal consisting of only zeros), nor it is defined for a signal of less than 2 elements."
-"An exception is thrown if the given envelope's size is not larger than 1. And also if the integral of the input envelope is 0 (i.e. envelope is only zeros or if its sum is 0).\n\n"
+"Please note that the TCToTotal ratio will return 0.5 for a zero signal (a signal consisting of only zeros) as 0.5 is the middle point of the signal. TCToTotal is not defined for a signal of less than 2 elements."
```

Comment:
I updated the tests so that they match this change in ecd3f5213e057194c56aa8aaa8f040e3401ff0c3
Also I fixed a bug in TCToTotal streaming that would not return 0.5 when signal is empty (and would return nan instead, an early return was missing).